lib/hammer_cli_foreman_templates/import.rb

```diff
@@ -5,9 +5,46 @@ class ImportCommand < HammerCLIForeman::Command
 
     resource :template, :import
 
-    success_message _('Import finished')
+    success_message _('Import finished.')
```

Comment:
Nit: why the `.` when `failure_message` does not have it?

Reply:
To keep consistency in messages refering to https://github.com/theforeman/hammer-cli/pull/266

Reply:
Ah, ok.